### DISCUSSION TITLE
sentinel限流次数  计数器  计数的数值是存储在哪里的？如果关掉sentinel服务在重启 是否次数重置还是有缓存继续以前的次数，如果有缓存，缓存存在哪里，有大佬解答一下吗？谢谢

### DISCUSSION THREAD

**Discussion by Li-enzhen** [alibaba/Sentinel#1945] [Q&A] [answered]
sentinel限流次数  计数器  计数的数值是存储在哪里的？如果关掉sentinel服务在重启 是否次数重置还是有缓存继续以前的次数，如果有缓存，缓存存在哪里，有大佬解答一下吗？谢谢

**Comment by sczyh30** [comment, marked as answer]
秒级数据仅存于内存中（秒级场景下缓存没有实际意义）

**Reply by Li-enzhen** [reply]
谢谢您！  还想请教一个问题是token server  单独部署的方式是单点的  如果我用pass平台  可以发现他挂掉了 并自动在起一个  这样可以避免回到原始的单示例限流吗？    token server 单独部署    咱官网是不是已经有实现 我只需要拉到拉到本地 启动就可以用了呀？

**Comment by Li-enzhen** [comment]
谢谢您！如果不是秒级的应用呢 存在哪里？ 还想请教一个问题是token server 单独部署的方式是单点的 如果我用pass平台 可以发现他挂掉了 并自动在起一个 这样可以避免回到原始的单示例限流吗？ token server 单独部署 咱官网是不是已经有实现 我只需要拉到拉到本地 启动就可以用了呀？恳请您指教一下 感谢!



在 2021-01-05 17:17:46，"Eric Zhao" <notifications@github.com> 写道：

秒级数据仅存于内存中（秒级场景下缓存没有实际意义）
